jobs/github/scripts/snippet_build_basic-make-check.sh

```diff
@@ -6,7 +6,7 @@
       # when running inside a privileged container, snapd fails because udevd isn't
       # running, but on the second occurance it is.
       # see: https://github.com/lxc/lxd/issues/4308
-      sudo snap refresh juju-db --channel=4.4 2> /dev/null || sudo snap install juju-db --channel=4.4
+      sudo snap refresh juju-db --channel=4.4 2> /dev/null || sudo snap install juju-db --channel=4.4 && echo "Installed:" && which juju-db.mongo && juju-db.mongo --version
```

Comment:
Is there a reason to do it as an && rather than just on the next line?

Reply:
Well I thought if the snap install fails then there's no reason to go further into `echo "Installed"` stuff, the right hand side of `&&` is only gonna execute if the snap stuff succeeds